js_functions_and_arrays/find_duplicate_values/index.js

```diff
@@ -0,0 +1,24 @@
+function findDuplicates(arr) {
+    const arrayOfDupes = [];
+    for (let i of arr) {
+        let count = 0;
+        for (let j of arr) {
+            if (i === j) {
+                count++;
+            }
+        }
+        if (count > 1) {
+            arrayOfDupes.push(i);
+        }
+        count = 0;
```

Comment:
You could have also checked if they are matched, then, if the empty array already does not include the match, push the match into the array.